### ISSUE TITLE
Agreed upon draw did not close match server side.

### ISSUE THREAD

**Issue by Zierman** [Zierman/JarChess#1]
Review blocked by the server side issue: https://github.com/atormanen/ICS499_ApplicationServer/issues/4

**Comment by Zierman** [comment]
@atormanen I'll review tomorrow after work, I have to get to bed tonight.

**Comment by Zierman** [comment]
I'm reviewing now

**Comment by Zierman** [comment]
~~I can't test because I am unable to log in (I expect login server is offline)... but~~ I was able to track down another issue (#19) and correct it due to that.

**Comment by Zierman** [comment]
I am still blocked, now because online matchmaking is not matching up players (see issue #21). I am starting to suspect that there are commented out lines of code in the app server to facilitate testing for issue #13.

**Comment by atormanen** [comment]
Apparently python doesn't allow for method overloading like java does... I changed the name of one of the methods server side inside game collection and match making is working again.

**Comment by Zierman** [comment]
> Apparently python doesn't allow for method overloading like java does... I changed the name of one of the methods server side inside game collection and match making is working again.

I’m glad that you shared that because it will probably save me from doing something similar down the road (^_^)... I’m a huge fan of method overloading.

**Comment by Zierman** [comment]
I can't test this to mark as done until we resolve https://github.com/atormanen/ICS499_ApplicationServer/issues/4

**Comment by atormanen** [comment]
Issue with version control. Server side code was not updated with the latest.

**Comment by Zierman** [comment]
Wonderful. When I tested it, a draw seems to successfully close a match. I'm closing this issue.
